link/perf_event_test.go

```diff
@@ -26,7 +26,7 @@ func TestTraceEventTypePMU(t *testing.T) {
 func TestTraceEventID(t *testing.T) {
 	c := qt.New(t)
 
-	eid, err := getTraceEventID("syscalls", "sys_enter_fork")
+	eid, err := getTraceEventID("syscalls", "sys_enter_execve")
```

Comment:
Why?

Reply:
From what I can tell `sys_enter_fork` was not available on arm, so I thought that for the purposes of testing `sys_enter_execve` is more portable.